src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -16,32 +16,211 @@ redirects:
+Setting up synthetic monitors with New Relic lets you:
 
-Synthetic monitoring has countless use cases, and knowing the right monitor type for yours is important. To learn about our different types of synthetic monitors and the differences between each, watch this short video (5:10 minutes).
+* Expand your browser monitoring with real, Selenium-powered [scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts), which test login procedures, searches, and other critical business transactions.
+* Run an [API test](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-api-tests) to check certificates or make sure any kind of HTTP requests are completed. 
+* Get extended use case coverage with [additional Node modules](/docs/synthetics/synthetic-monitoring/scripting-monitors/import-nodejs-modules) for your scripted API monitors. 
+* Easily diagnose whether an issue stems from the network or AWS location, a slow third party resource, or the health of your backend services or infrastructure. 
+
+For a short demo of how to create synthetic monitors with New Relic, watch the below video (5:10 minutes).
 
 <Video
   id="EN5FkIW8YWk"
   type="youtube"
 />
 
-## Advanced testing [#advanced]
+## Before you create your first synthetic monitor...
+
+Once you've [signed up for a New Relic account](https://newrelic.com/signup), you're ready to create your first synthetic monitor. Below, you can get a high-level overview of system requirements, how we protect data, and how you can segment monitoring capabilities by roles within an organization.     
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="compatibility"
+    title="Are there any system requirements with synthetic monitoring?"
+  >
+Synthetic monitoring does not require any software except a [supported browser](/docs/apm/new-relic-apm/getting-started/supported-browsers).
+
+<Callout variant="important">
+  To monitor a site behind your firewall, add the [synthetic monitoring public
+  minion IP
+  addresses](/docs/synthetics/new-relic-synthetics/using-monitors/synthetics-public-minion-ips)
+  to your allow list or [create a private location](/docs/synthetics/synthetic-monitoring/private-locations/private-locations-overview-monitor-internal-sites-add-new-locations/).
+</Callout>
+
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="privacy"
+    title="What kind of data do synthetic monitors create?"
+  >
+The data from synthetic monitoring is test data, representing typical interaction with the webpage or application. It is never actual data from human beings. The data collected when you use synthetic monitoring therefore is not personal data. For more information, see the [Synthetic monitoring security documentation](/docs/synthetics/new-relic-synthetics/getting-started/security-new-relic-synthetics).
```

Comment:
When I read this the "therefore" kind of stops my flow. Not a big deal either way, just something to consider.